libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -7,16 +8,41 @@
 
 extern const AP_HAL::HAL& hal;
 
+const int16_t POI_REQ_TIMOUT_MS = 3000;     // requests over 3 seconds old are ignored
+
 // Constructor
 AP_Camera_Backend::AP_Camera_Backend(AP_Camera &frontend, AP_Camera_Params &params, uint8_t instance) :
     _frontend(frontend),
     _params(params),
     _instance(instance)
 {}
 
+// common initialisation for all backends
+void AP_Camera_Backend::init()
+{
+#if AP_TERRAIN_AVAILABLE
+    // create a calculation thread for poi.
+    if (!hal.scheduler->thread_create(FUNCTOR_BIND_MEMBER(&AP_Camera_Backend::calculate_poi, void),
+                                      "poi_calculation_thread",
+                                      8192, AP_HAL::Scheduler::PRIORITY_IO, -1)) {
```

Comment:
Check this thread priority - is it too high?  Too low?  Why did you choose IO?

Reply:
To resolve this could you suggest me the way to check the best stack size and priority ?